_employment/employers/commitments.md

```diff
@@ -21,16 +21,15 @@ template: 1-topic-landing
 
 Employers from every industry and in every state have committed to hiring Veterans and military family members because these individuals have consistently proven themselves to be reliable, independent, and hardworking. 
 
-By signing up and [making a commitment](/_dummy-placeholder.html), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html). 
+By signing up and [making a commitment](/employment/users/sign_in), your company will stand out to candidates eager to enter the civilian workforce. There are also tax benefits to hiring Veterans, such as the Work Opportunity Tax Credit (WOTC), that range from $1,200 to $9,600. Other VA programs provide incentives to hiring Veterans, such as  [Vocational Rehabilitation and Employment](/_dummy-placeholder.html ) and [On-the-Job Training](/_dummy-placeholder.html). 
```

Comment:
@mollyblake do we want to link to the commitments page from here instead of the sign in page? (I am not sure)